axelor-base/src/main/java/com/axelor/apps/base/service/administration/SequenceService.java

```diff
@@ -121,11 +121,17 @@ public static boolean isMonthValid(Sequence sequence) {
         && (seq.contains(PATTERN_YEAR) || seq.contains(PATTERN_FULL_YEAR));
   }
 
-  public static boolean isSequenceLengthValid(Sequence sequence) {
-    String seqPrefixe = StringUtils.defaultString(sequence.getPrefixe(), "").replace("%", "");
-    String seqSuffixe = StringUtils.defaultString(sequence.getSuffixe(), "").replace("%", "");
+  public void isSequenceLengthValid(Sequence sequence) throws AxelorException {
```

Comment:
please rename the method `checkSequenceLengthValidity` since it does not return a boolean anymore